### ISSUE TITLE
New iOS App doesn't update item status

### ISSUE THREAD

**Issue by manroh** [openhab/openhab1-addons#1187]
I just installed the new app version which is very nice with the new interface.
Only drawback is, that item status for instance light on/off is not automatically updated while changed. The page has to be reloaded first. This worked with the old verision.

**Comment by bodomenke** [comment]
Is this fixed in the meanwhile? Reason for asking is that I have issue with the iOS app not refreshing status after staying approx. 10 to 15 min on a page.

**Comment by mabakach** [comment]
For me it looks like this still not working. I use openHAB 1.7.1 and version 1.7.1 of the iOS App. Until now I had this only once and a restart of the openHAB server fixed it for the moment.

**Comment by bodomenke** [comment]
See the thread in the openHab community:
https://community.openhab.org/t/refresh-of-ios-app/3941

**Comment by awnz** [comment]
I'm getting some weird related behaviour in the iOS App.

Sub-pages won't update for me if they have a switch in them, unless I also define a switch (any switch) above it (not below it) on the main page - then they work fine. The switch of course can also be a button.

So updates for sub-pages work for me in my case when I defined an "All Off" switch right at the top first. That'll do me so unless I get curious I won't do any further workaround testing, but it might be useful for others as a starting point.
